jstz_cli/src/node.rs

```diff
@@ -0,0 +1,58 @@
+use std::time::Duration;
+
+use anyhow::Result;
+use jstz_proto::{operation::OperationHash, receipt::Receipt};
+use tokio::time::sleep;
+
+pub struct JstzNode {
```

Comment:
Should this be part of jstz_cli? Wouldn't it be better in jstz_node with a name like e.g. JstzNodeReader?